### ISSUE TITLE
error: ‘AVFormatContext {aka struct AVFormatContext}’ has no member named ‘url’

### ISSUE THREAD

**Issue by ysig** [sportdeath/audiorw#2]
I am trying to install audio transport on my computer and by doing so, I landed to this repo.
I am having a problem to install audiorw, due to the following error:

```
$ make
Scanning dependencies of target audiorw
[ 33%] Building CXX object CMakeFiles/audiorw.dir/src/read.cpp.o
[ 66%] Building CXX object CMakeFiles/audiorw.dir/src/write.cpp.o
/home/syro/mirage/Desktop/audiorw/src/write.cpp: In function ‘void audiorw::write(const std::vector<std::vector<double> >&, const string&, double)’:
/home/syro/mirage/Desktop/audiorw/src/write.cpp:63:27: error: ‘AVFormatContext {aka struct AVFormatContext}’ has no member named ‘ur ’
   if (!(format_context -> url = av_strdup(filename.c_str()))) {
                           ^~~
CMakeFiles/audiorw.dir/build.make:86: recipe for target 'CMakeFiles/audiorw.dir/src/write.cpp.o' failed
make[2]: *** [CMakeFiles/audiorw.dir/src/write.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/audiorw.dir/all' failed
make[1]: *** [CMakeFiles/audiorw.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```
This is possibly related with ffmpeg.
The version I am using is `ffmpeg version 3.4.8`.
What version should I use to install your package.

Thanks in advance,

**Comment by ysig** [comment]
I solved it by updating ffmpeg to ffmpeg-4.

**Comment by ysig** [comment]
Fixed with ffmpeg version 4.0.
